server/src/server.js

```diff
@@ -219,7 +219,17 @@ app.use(function(req, res, next) {
     req.accountId = authInfo.accountId;
   }
   req.cookies = cookies;
-  req.cookies._csrf = cookies.get("_csrf"); // csurf expects a property
+  // The cookies library doesn't detect duplicates; check manually
+  let rawCookies = req.get("cookie") || "";
+  let pairs = rawCookies.split(";");
+  let csrfTokens = pairs.filter(item => { return item.match(/__Host-csrf=/); });
```

Comment:
This also matches cookies like "foo__Host-csrf=", but that shouldn't matter since duplicates are handled below.